### PR TITLE
[Doc] update supported security measures

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: docs/sql-reference/sql-statements/data-manipulation/ROUTINE LOAD.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Routine Load allows you to submit a permanent import task to import data into StarRocks by constantly reading from a specified data source. Currently only support importing text format (CSV) data from Kafka without authentication or SSL authentication.
+Routine Load allows you to submit a permanent import task to import data into StarRocks by constantly reading from a specified data source. Currently Routine Load supports importing text format (CSV) data from Kafka without authentication，encryption and authentication using SSL, as well as authentication using SASL.
```
Comment (Dshadowzh) [inline docs/sql-reference/sql-statements/data-manipulation/ROUTINE LOAD.md:5]: Not only CSV, Json is supported. And I suggestion separate it into two sentence, one for file format, one for authentication and authentication

**Review by Dshadowzh — APPROVED** [review]
LGTM

**Review by femiiii — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by hellolilyliuyi** [comment]
run starrocks_be_unittest

**Pull request merged**